include/SFML/Window/Keyboard.hpp

```diff
@@ -145,9 +153,205 @@ enum Key
     F15,          //!< The F15 key
     Pause,        //!< The Pause key
 
-    KeyCount //!< Keep last -- the total number of keyboard keys
+    KeyCount, //!< Keep last -- the total number of keyboard keys
+
+    // Deprecated values:
+
+    Tilde     = Grave,     //!< \deprecated Use Grave instead
+    Dash      = Hyphen,    //!< \deprecated Use Hyphen instead
+    BackSpace = Backspace, //!< \deprecated Use Backspace instead
+    BackSlash = Backslash, //!< \deprecated Use Backslash instead
+    SemiColon = Semicolon, //!< \deprecated Use Semicolon instead
+    Return    = Enter,     //!< \deprecated Use Enter instead
+    Quote     = Apostrophe //!< \deprecated Use Apostrophe instead
+};
+
+////////////////////////////////////////////////////////////
+/// \brief Scancodes
+///
+/// The enumerators are bound to a physical key and do not depend on
+/// the keyboard layout used by the operating system. Usually, the AT-101
+/// keyboard can be used as reference for the physical position of the keys.
+///
+////////////////////////////////////////////////////////////
+struct Scan
```

Comment:
This one is a bit confusing to me. In some places we refer to these enumerations like `Scan::A` but elsewhere we refer to the _type_ of the enumeration as `Scancode`. I'd like to replace this with a scoped enumeration but I'm not sure how to resolve both different ways we refer to scan codes.

Reply:
I was suggesting a namespace because we recently replaced structs with no non-static members by namespaces.
However, in the case of `Scan`, it is intended to eventually become a scoped enumeration so we might as well do it right away.
Regarding `Scan` vs `Scancode`, it can simply be a type alias.